evm_arithmetization/src/witness/util.rs

```diff
@@ -305,7 +316,7 @@ pub(crate) fn keccak_sponge_log<F: Field, T: Transition<F>>(
             address.increment();
         }
         xor_into_sponge::<F, _>(state, &mut sponge_state, block.try_into().unwrap());
-        state.push_keccak_bytes(sponge_state, clock * NUM_CHANNELS);
+        state.push_keccak_bytes(sponge_state, (clock - 1) * NUM_CHANNELS + 1);
```

Comment:
All these `(clock - 1)` could disappear if the `get_clock()` method was updated directly to account for the offset